src/utilities/permissions.ts

```diff
@@ -348,6 +351,15 @@ const queryPermissions: Record<GQLKeys, (user: User | null, ...args: any[]) => b
     const queries = [Queries.CREATE_EXPANSION_SET];
     return isUserAdmin(user) || (getPermission(queries, user) && getRoleModelPermission(queries, user, plans, model));
   },
+  CREATE_EXTERNAL_EVENT_TYPE: (user: User | null): boolean => {
+    return isUserAdmin(user) || getPermission([Queries.EXTERNAL_EVENT_TYPES], user);
```

Comment:
And I think this needs to be a `insert_external_event_type_one` query, but I'm not seeing it as a part of the `Queries` enum